app/dashboard/verify/page.tsx

```diff
@@ -1,26 +1,26 @@
 'use client';
 
-import { getAuthenticatedUser, useSelector } from "@/lib/redux";
+import { fetchDJs, getAuthenticatedUser, useDispatch, useSelector } from "@/lib/redux";
 import { getter } from "@/lib/services";
 import { useEffect } from "react";
 import { toast } from "sonner";
 
 export default function VerifyPage() {
 
+  const user = useSelector(getAuthenticatedUser);
+  const dispatch = useDispatch();
+
   useEffect(() => {
     console.log("mounted");
 
     const process = async (e: KeyboardEvent) => {
```

Comment:
Could you rename this to something more descriptive like `processKeyboardInput` or something?